app/uk/gov/hmrc/cataloguefrontend/connector/model/Dependencies.scala

```diff
@@ -363,7 +363,9 @@ object ServiceWithDependency {
 case class GroupArtefacts(
   group    : String,
   artefacts: List[String]
-)
+) {
+  def searchStrings: Seq[String] = artefacts.map(artefact => s"$group:$artefact")
```

Comment:
Feels like half of this is in the view (`allGroupArtefacts`) and the other in the model. Can we move it all to the view?